diff --git a/build/modules.conf.in b/build/modules.conf.in
index 9b2e360811..df59f0fe8e 100755
--- a/build/modules.conf.in
+++ b/build/modules.conf.in
@@ -32,7 +32,7 @@ applications/mod_httapi
 #applications/mod_osp
 #applications/mod_prefix
 #applications/mod_redis
-applications/mod_signalwire
+#applications/mod_signalwire
 applications/mod_sms
 #applications/mod_snapshot
 applications/mod_spandsp
